src/test/java/net/fornwall/jelf/ElfParserTest.java

```diff
@@ -0,0 +1,133 @@
+package net.fornwall.jelf;
+
+import com.diffblue.deeptestutils.Reflector;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
```

Comment:
I don't think this is needed?

Reply:
Method `testRead()` throws a IOException